src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -64,7 +64,8 @@ namespace
             m_background_emits_light = true;
             m_background_alpha = 0.0f;
             m_force_off_default_lights = false;
-
+			m_clamp_roughness = true;
```

Comment:
As this is a hack, it should be off by default (just like Max Ray Intensity).

Reply:
Place it next to Max Ray Intensity as they belong to the same category (performance hacks).